.github/workflows/docs.yml

```diff
@@ -91,17 +91,6 @@ jobs:
           path: moabb-ghio
           token: ${{ secrets.MOABB_GHIO }}
 
-      - name: Deploy on moabb.neurotechx.com
-        run: |
-          git config --global user.email "ci@neurotechx.com"
-          git config --global user.name "Github Actions"
-          cd ~/work/moabb/moabb/moabb-ghio
-          rm -Rf docs
-          cp -a ~/work/moabb/moabb/docs/build/html ./docs
-          git add -A
-          git commit -m "GH Actions update of docs ($GITHUB_RUN_ID - $GITHUB_RUN_NUMBER)"
-          git push origin master
-
   deploy_gh_pages:
     if: ${{ github.ref == 'refs/heads/develop' }}
```

Comment:
It seems to work, maybe we could a different approach to use `workflow_run`. It means that we should split the docs.yml into docs.yaml and docs-deploy.yml, with deployement actions moved in docs-deploy.yml with conditional triggers requiring that the doc building is completed. There is an [example on SO.](https://stackoverflow.com/a/69743847)

Reply:
Can we optimize this flow later?

Reply:
Yes